src/aznfswatchdog

```diff
@@ -138,45 +140,96 @@ reconcile_conntrack()
     done
 }
 
-vecho "Starting aznfswatchdog..."
 
-# Detect and log distro and bash version
-if [ -f /etc/centos-release ]; then
-    linux_distro=$(cat /etc/centos-release 2>&1)
-elif [ -f /etc/os-release ]; then
-    linux_distro=$(grep "^PRETTY_NAME=" /etc/os-release | awk -F= '{print $2}' | tr -d '"')
-else
-    linux_distro="Unknown"
-fi
+#
+# Delete entry from $AZ_FILES_MOUNTMAP.
+#
+ensure_az_files_mountmap_not_exist()
+{
+    (
+        flock -e 999
 
-bash_version=$(bash --version | head -n 1)
-vecho "Linux distribution: $linux_distro"
-vecho "Bash version: $bash_version"
+        #
+        # If user wants to delete the entry only if AZ_FILES_MOUNTMAP has not changed since
```

Comment:
`I *user* wants`, what is the user here mean? or you mean the watchdog?

Reply:
yes, i mean watchdog